pkg/ccl/streamingccl/streamingest/stream_replication_e2e_test.go

```diff
@@ -1278,6 +1282,6 @@ func TestTenantStreamingShowTenant(t *testing.T) {
 	require.Less(t, maxReplTime, timeutil.Now())
 	require.Less(t, protectedTime, timeutil.Now())
 	require.GreaterOrEqual(t, maxReplTime, highWatermark.GoTime())
-	// TODO(lidor): replace this start time with the actual replication start time when we have it.
-	require.GreaterOrEqual(t, protectedTime, testStartTime)
+	require.GreaterOrEqual(t, protectedTime, replicationDetails.ReplicationStartTime.GoTime())
+	require.Equal(t, replicationStartTime.UnixNano(), replicationDetails.ReplicationStartTime.GoTime().UnixNano())
```

Comment:
can you drop the `UnixNano()`?

Reply:
The test flake actually revealed that the `replicationStartTime` is only accurate up to the microsecond after we start running into rounding mismatches. I had to change this to `UnixMicro` to pass under stress on a gceworker.